test/runtests.jl

```diff
@@ -1,41 +1,67 @@
 using Test
 using DFTK
 
+run_expensive = true  # Run expensive tests or not?
```

Comment:
the logic here is a bit hard to follow, maybe add a comment describing the conditions under which each type of tests are run?

Reply:
You're right, the logic is a bit convoluted. I'll try to make that clearer and add some comments.